lib/metacat.properties

```diff
@@ -190,46 +190,11 @@ database.indexingInitialDelay=3600000
 database.maximumIndexDelay=5000
 database.runDBConnectionRecycleThread=off
 database.cycleTimeOfDBConnection=30000
-database.queryignoredparams=enableediting,foo
-database.usexmlindex=true
-# used for the setting the size of resultset for applications like morpho
-database.appResultsetSize=7000
 # used for the setting the size of resultset for searches done using browsers
 database.webResultsetSize=7000
-# the value of xml_returnfield.usage_count should be more than this value
-# for records to be entered into xml_queryresult. so if you want results for
-# any combination of returnfields to be stored in xml_queryresult only when
-# that combination has been requested 50 times, set this value to 50
-database.xmlReturnfieldCount=0
-# used for the setting the size of queryresult_string in queryresult table.
-# the limit is 4000 for oracle
-database.queryresultStringLength=500000
-#the size of query result cache
-database.queryresultCacheSize=500
-#turn on or off the query result cache
-database.queryCacheOn=true
-#the time in milliseconds that an squery can run before metacat logs a warning
-database.queryTimeWarnLimit=30000
-#the time in milliseconds that an squery can run before metacat logs a warning
-database.squeryTimeWarnLimit=30000
-
 
 ######## DB Query section              #######################################
 dbquery.enabledEngines=solr
```

Comment:
see [this comment](https://github.com/NCEAS/metacat/pull/1726/files#diff-b255420e6f8839b7c622e62c3f859c25039069a7ad5dfae1bd8c07061ba7ae40R2207), further down the page